tests/cases/case_providers_care_platform.yaml

```diff
@@ -0,0 +1,441 @@
+      Id: $PAGE.care_platform_call_portal.issue_resolved_yes_radiobutton$          
+    - Type: click
+      Strategy: id
+      Id: $PAGE.care_platform_call_portal.category_select$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.medical_question_option$
+    - Type: click
+      Strategy: id
+      Id: $PAGE.care_platform_call_portal.subcategory_select$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.located_provider_option$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.submit_and_exit_session_button$
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.care_platform_notifications.call_completed_notification$
+      # CXTB-217
+HomePageListingView:
+  Environment: Settings 
+  Roles:
+    - Role: desktopChrome
+      Capabilities:
+        chromeOptions:
+          args:
+            - use-fake-ui-for-media-stream
+            - use-fake-device-for-media-stream
+            - no-sandbox
+            - disable-gpu
+      App: desktop
+    - Role: localiOS
+      App: NeverAlone
+  Actions:
+      # Login to the Care Platform app with Care Partners account
+    - Type: case
+      Value: CarePartnerLogin
+      # Setup for the test data
+      # Missed call 
+    - Type: case
+      Value: set_missed_call_ticket
+    - Type: case
+      Value: TerminateApp
+      # Logout from the Care Partner account
+    - Type: case
+      Value: care_platform_logout
+      # Login to the app with Providers account
+    - Type: case
+      Value: providers_care_platform_login
+      # Follow-up ticket
+    - Type: case
+      Value: LaunchApp
+    - Type: case
+      Value: set_follow_up_ticket
+      # Logout from the Care Partner account
+    - Type: case
+      Value: care_platform_logout
+      # Login to the Care Platform app with Care Partners account
+    - Type: case
+      Value: CarePartnerLogin
+      # Scheduled call
+    - Type: case
+      Value: set_scheduled_call_ticket
+      # Logout from the Care Partner account
+    - Type: case
+      Value: care_platform_logout
+      # Login to the app with Providers account
+    - Type: case
+      Value: providers_care_platform_login
+      # Set the Time to All
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.providers_home_page.time_dropdown$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.providers_home_page.time_all_option$
+    - # Check that there are no active calls waiting to be answered
+      Type: wait_not_visible
+      Strategy: xpath
+      Id: $PAGE.providers_home_page.last_medical_transfer_row$  
+    - Type: wait_not_visible
+      Strategy: xpath
+      Id: $PAGE.providers_home_page.last_on_demand_row$  
+      # Verify Missed calls are present
+    - Type: get_text  
+      Strategy: xpath
+      Id: $PAGE.providers_home_page.missed_calls_text$
+      Greps:
+      - var: missed_call_text
+        match: "(.*)"
+        NoRaise: true
```

Comment:
is this NoRaise set to true on purpose?

Reply:
they were no longer necessary, I removed them